src/layout/default.hbs

```diff
@@ -9,5 +9,9 @@
     {{!-- Body conent is loaded in this area of the template,
           these are the contents of each pages/*.hbs file --}}
     {{{body}}}
+    <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
```

Comment:
I needed this line for some of my Javascript to work. Nate helped me with it, and I got that whole script from here: https://code.jquery.com/.